From 3ee975a1cf60e0c8791ad4ecdf77eb91d8f650b8 Mon Sep 17 00:00:00 2001
From: Ding Wei <leo.ding@rock-chips.com>
Date: Fri, 29 Jan 2021 16:14:55 +0800
Subject: [PATCH] video: rockchip: mpp: control the log, when task is null

tips:
    in rk3328, avsd and vdpu share the same interrupt number,
    so, when each one hardware done, avsd and vdpu irq will response.
    this is a normal log when meet the case.

Change-Id: I9fea220a64d7310805194d9f71c24db0cef1491e
Signed-off-by: Ding Wei <leo.ding@rock-chips.com>
---
 drivers/video/rockchip/mpp/mpp_common.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/rockchip/mpp/mpp_common.c b/drivers/video/rockchip/mpp/mpp_common.c
index 41086c2bbb18..87b48c60fe97 100644
--- a/drivers/video/rockchip/mpp/mpp_common.c
+++ b/drivers/video/rockchip/mpp/mpp_common.c
@@ -1751,7 +1751,7 @@ irqreturn_t mpp_dev_irq(int irq, void *param)
 			set_bit(TASK_STATE_IRQ, &task->state);
 		}
 	} else {
-		mpp_err("error, task is null\n");
+		mpp_debug(DEBUG_IRQ_CHECK, "error, task is null\n");
 	}
 done:
 	return irq_ret;
-- 
2.35.3

